flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/sink/TestCompressionSettings.java

```diff
@@ -91,19 +94,36 @@ public void testCompressionAvro() throws Exception {
             ImmutableMap.of(FlinkWriteOptions.WRITE_FORMAT.key(), "AVRO"));
 
     if (initProperties.get(TableProperties.AVRO_COMPRESSION) == null) {
-      Assert.assertEquals(
-          TableProperties.AVRO_COMPRESSION_DEFAULT,
-          resultProperties.get(TableProperties.AVRO_COMPRESSION));
-      Assert.assertEquals(
-          TableProperties.AVRO_COMPRESSION_LEVEL_DEFAULT,
-          resultProperties.get(TableProperties.AVRO_COMPRESSION_LEVEL));
+      assertThat(resultProperties)
+          .containsEntry(TableProperties.AVRO_COMPRESSION, TableProperties.AVRO_COMPRESSION_DEFAULT)
+          .satisfies(
+              props -> {
+                if (props.containsKey(TableProperties.AVRO_COMPRESSION_LEVEL_DEFAULT)) {
```

Comment:
I don't fully follow this assertion here. Isn't this checking whether the map contains a null key because `TableProperties.AVRO_COMPRESSION_LEVEL_DEFAULT` defaults to null?

Reply:
Sorry, I'll fix this part. As you're mentioning, this part checks if the compression is null or not null based on a specified compression format.